src/main/java/com/fauna/codec/UTF8FaunaParser.java

```diff
@@ -9,11 +9,13 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.format.DateTimeParseException;
 import java.util.*;
```

Comment:
maybe a good time to expand this if you'd like

Reply:
I think you mean the `util.*` and I agree.